stories/core/Toasts.stories.tsx

```diff
@@ -73,17 +97,25 @@ export default {
     },
   },
   title: 'Core/Toasts',
-} as Meta<ToastProps> & CreeveyMeta;
+} as Meta<ToastProps & ToasterSettings> & CreeveyMeta;
 
-export const Positive: Story<ToastProps> = ({
+export const Positive: Story<ToastProps & ToasterSettings> = ({
   content,
   duration,
   hasCloseButton,
   link,
   type,
   onRemove,
+  placement,
+  order,
 }) => {
+  const settings: ToasterSettings = {
```

Comment:
So I just realized that this `ToasterSettings` type is not exported in the barrell. Maybe don't declare settings explicitly then. Instead, you can just inline this value in line 118